document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AbridgedAccountsDataHandler.java

```diff
@@ -166,27 +158,15 @@ private String createPathString(AccountType accountType) {
         return account.toString();
     }
 
-    /**
-     * Get the period end Date from the account data
-     *
-     * @param accountData the account data
-     * @return periodEndDate The formatted periodEndDate
-     * @throws ParseException
-     */
-    private <T> Date getCurrentPeriodEndOn(T accountData) throws ParseException {
+    private Date getCurrentPeriodEndOn(AbridgedAccountsApi accountData) throws ParseException {
```

Comment:
Same here, Can we use LocalDate, please?

Reply:
done